OutOfSchool/OutOfSchool.WebApi/Services/ParentService.cs

```diff
@@ -147,5 +179,29 @@ public async Task<ShortUserDto> Update(ShortUserDto dto)
                 throw;
             }
         }
+
+        private Expression<Func<Parent, bool>> PredicateBuild(SearchStringFilter filter)
+        {
+            var predicate = PredicateBuilder.True<Parent>();
+
+            if (!string.IsNullOrWhiteSpace(filter.SearchString))
+            {
+                var tempPredicate = PredicateBuilder.False<Parent>();
+
+                foreach (var word in filter.SearchString.Split(' ', ',', StringSplitOptions.RemoveEmptyEntries))
+                {
+                    tempPredicate = tempPredicate.Or(
+                        x => x.User.FirstName.StartsWith(word, StringComparison.InvariantCulture)
```

Comment:
You can add the extension method to the string type, smth like a InvariantStartWith

Reply:
I can't do it right now. There is an exception when i use extension method. But I'll try to find the solution for this.

Reply:
Could you please show a small code example when the exception is thrown ?